nni/retiarii/nn/pytorch/nn.py

```diff
@@ -108,24 +98,141 @@ def wrap_module(original_class):
     # Make copy of original __init__, so we can call it without recursion
 
     def __init__(self, *args, **kws):
-        global _records
-        if _records is not None:
-            full_args = {}
-            full_args.update(kws)
-            for i, arg in enumerate(args):
-                full_args[argname_list[i]] = args[i]
-            _records[id(self)] = full_args
+        full_args = {}
+        full_args.update(kws)
+        for i, arg in enumerate(args):
+            full_args[argname_list[i]] = args[i]
+        add_record(id(self), full_args)
+
         orig_init(self, *args, **kws) # Call the original __init__
 
     original_class.__init__ = __init__ # Set the class' __init__ to the new one
     return original_class
 
-Conv2d = wrap_module(nn.Conv2d)
+'''Conv2d = wrap_module(nn.Conv2d)
 BatchNorm2d = wrap_module(nn.BatchNorm2d)
 ReLU = wrap_module(nn.ReLU)
 Dropout = wrap_module(nn.Dropout)
 Linear = wrap_module(nn.Linear)
 MaxPool2d = wrap_module(nn.MaxPool2d)
 AvgPool2d = wrap_module(nn.AvgPool2d)
 Identity = wrap_module(nn.Identity)
+AdaptiveAvgPool2d = wrap_module(nn.AdaptiveAvgPool2d)'''
+
+Identity = wrap_module(nn.Identity)
```

Comment:
Can we refactor it into a for loop?

Reply:
i think this way is more clear and easy to maintain

Reply:
because not all the modules are included, for example, `loss` modules. and different versions of pytorch have different members in nn.module. I will handle different versions later